tests/functional_tests/iterator_tests/test_toy_connectivity.py

```diff
@@ -110,15 +110,15 @@ def sum_edges_to_vertices(in_edges):
     )
 
 
-def test_sum_edges_to_vertices(fencil_processor):
-    fencil_processor, validate = fencil_processor
+def test_sum_edges_to_vertices(fencil_processor_no_gtfn_exec):
+    fencil_processor_no_gtfn_exec, validate = fencil_processor_no_gtfn_exec
```

Comment:
```suggestion
    fencil_processor, validate = fencil_processor_no_gtfn_exec
```
It's much easier to activate the tests again if the name `fencil_processor` stays. I'll skip the comments below.